self-core-impl/src/main/java/com/selfxdsd/core/tasks/ContributorTasks.java

```diff
@@ -100,6 +100,12 @@ public Tasks ofContributor(final String username,
         return tasks;
     }
 
+    @Override
+    public Tasks unassigned() {
+        throw new UnsupportedOperationException("Contributors can't have "
+            + " unassigned tasks");
```

Comment:
@criske Can you give a more detailed message: "These are the Tasks or Contributor X from provider Y, no unassigned Tasks here.".